tests_e2e/orchestrator/sample_runbooks/existing_vm.yml

```diff
@@ -61,5 +85,11 @@ notifier:
   - type: env_stats
   - type: agent.junit
 
-include:
-  - path: ../include/ssh_proxy.yml
```

Comment:
there is no need for a separate file, since we are using a single runbook. the other runbooks (exiting vm and local are samples for the developer scenario)